### ISSUE TITLE
feat: Display the latest_version_timestamp for each project

### ISSUE THREAD

**Issue by maxlerebourg** [release-argus/Argus#382]
**Is your feature request related to a problem? Please describe.**
I want to see if the projects I monitor is currently under development.

**Describe the solution you'd like**
Display on the homepage of Argus, for each project the `latest_version_timestamp`.

**Additional context**
I took inspiration for this feature from https://selfh.st/apps/
Maybe have a color code for long inactivity period like there

**Comment by JosephKav** [comment]
You can view the `latest_version_timestamp` by hovering over the latest version
<img width="258" alt="image" src="https://github.com/release-argus/Argus/assets/4267227/8fa74815-2405-4ea9-a458-b53bbe536cb3">

Not sure about colour coding this as some projects do release infrequently and I don't think this should always draw more attention

**Comment by maxlerebourg** [comment]
<img width="737" alt="image" src="https://github.com/release-argus/Argus/assets/22881669/5612b2bc-a751-429f-9c0b-b0207054c8a2">
This is the `deployed_version_timestamp` not the `latest_version_timestamp`.
If a project is "dead", I want to know it even without color code, but a little "latest update: ..." could be very useful.

**Comment by maxlerebourg** [comment]
I agree with the toggle, but if you can display it without hover the deployed version, it will be helpful.

**Comment by JosephKav** [comment]
> display it without hover the deployed version

Sorry, what do you mean by this?
"Current version" is the deployed version if that matches the latest version. Otherwise this would be displayed
<img width="337" alt="image" src="https://github.com/release-argus/Argus/assets/4267227/5ae1b6f4-7bec-48e8-a376-04c0d67e4609">
Which can also be hovered for the dates
<img width="338" alt="image" src="https://github.com/release-argus/Argus/assets/4267227/1419e16e-3db2-4daa-b1d8-567dfc337468">

**Comment by JosephKav** [comment]
That was the point of this verbose/more info button. Don't want to make the cards bigger for the date as most people won't regularly need it

**Comment by maxlerebourg** [comment]
Awesome !! Yes totally agree.
